tests/product/test_server_install.py

```diff
@@ -445,10 +441,21 @@ def test_install_non_root_user(self):
         rpm_name = installer.copy_presto_rpm_to_master(cluster=self.cluster)
         self.write_test_configs(self.cluster)
         self.run_prestoadmin(
-            'server install ' + os.path.join(self.cluster.mount_dir, rpm_name) + ' -p password'
+            'server install {rpm_dir}/{name} -p password'.format(
+                rpm_dir=self.cluster.get_rpm_cache_dir(), name=rpm_name)
         )
 
         for container in self.cluster.all_hosts():
             installer.assert_installed(self, container)
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
+
+    @staticmethod
+    def format_err_msgs_with_internal_hosts(cluster, msgs):
```

Comment:
No good reason really. I was expecting to re-use the method in other classes. I'll make it an instance method.